export-helpers/background.js

```diff
@@ -0,0 +1,30 @@
+function onExecuted() {
+  if (chrome.runtime.lastError) {
+    console.log(chrome.runtime.lastError);
+  }
+}
+
+/*
+Execute content script in the active tab.
```

Comment:
Nit, we could indent the inline comment content to make them easier to be spotted.